bokehjs/test/unit/regressions.ts

```diff
@@ -1744,4 +1744,26 @@ describe("Bug", () => {
       expect(getComputedStyle(pv3.canvas.events_el).touchAction).to.be.equal("none")
     })
   })
+
+  describe("in issue #14164", () => {
+    it("doesn't allow to use the correct CSS color syntax in SVG output", async () => {
+      const p = fig([200, 200], {output_backend: "svg"})
+      p.scatter([1, 2, 3], [1, 2, 3], {
+        size: 20,
+        color: ["rgb(14 15 16)", "rgb(17 18 19 / 0.877)", "rgba(20, 21, 22, 0.577)"],
+      })
+
+      const {view} = await display(p)
+      const blob = await view.export().to_blob()
+      const svg = await blob.text()
+
+      expect(svg.includes('"rgb(14 15 16)"')).to.be.false
+      expect(svg.includes('"rgb(17 18 19 / 0.877)"')).to.be.false
+      expect(svg.includes('"rgba(20, 21, 22, 0.577)"')).to.be.false
+
+      expect(svg.includes('"rgb(14,15,16)"')).to.be.true
+      expect(svg.includes('"rgb(17,18,19)"')).to.be.true
+      expect(svg.includes('"rgb(20,21,22)"')).to.be.true
```

Comment:
test for separate opacity?

Reply:
I replaced the test with a more robust one.